src/objects.jl

```diff
@@ -45,6 +45,8 @@ type LinearSPModel <: SPModel
     equalityConstraints::Union{Void, Function}
     inequalityConstraints::Union{Void, Function}
 
+    refTrajectories::Union{Void, Array{Float64, 3}}
```

Comment:
That can be only temporary : an SPmodel cannot contain a "refTrajectories" which only represent an algorithm dependant property and not a stochastic model property.